cargo-miri/bin.rs

```diff
@@ -398,22 +466,26 @@ fn in_cargo_miri() {
         return;
     }
 
+    let (target_filters, cargo_args, miri_args) =
```

Comment:
```suggestion
    // FIXME: this accepts --test, --lib and multiple --bin for `cargo miri run`.
    let (target_filters, cargo_args, miri_args) =
```
I think it's okay to accept this for now, but we should leave a comment reminding us of the problem.